logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SlowSyncWorker.kt

```diff
@@ -18,6 +19,7 @@ class SlowSyncWorker(
         kaliumLogger.d("Sync: Starting SlowSync")
 
         val result = try{
+            userSessionScope.getRemoteFeatureConfigsStatusAndPersist
```

Comment:
Shouldn't we invoke `()` on this?

`userSessionScope.getRemoteFeatureConfigsStatusAndPersist()` ?

Reply:
yeah , already add it , my bad